tox.ini

```
@@ -15,15 +15,10 @@ passenv =
   TRAVIS_JOB_NUMBER
   TRAVIS_PULL_REQUEST
   TRAVIS_REPO_SLUG
-deps =
-    pytest
-    freezegun
-    !py36: vdirsyncer
-    py36: vdirsyncer==0.16.8
-    python-dateutil
+extras =
+    tests
     pytz20187: pytz==2018.7
     pytz_latest: pytz
```

These should remain in `deps =`:

```suggestion
deps =
    pytz20187: pytz==2018.7
    pytz_latest: pytz
```